ctags/parsers/python.c

```diff
@@ -27,6 +27,17 @@
 /*
 *   DATA DEFINITIONS
 */
+
+struct corkPair {
+	int index;
+};
```

Comment:
I kept the struct, just renamed it to corkInfo (int corkIndex is already used at various places in functions and I couldn't think of any good name). Will be replaced by the new parser anyway.